qiskit/transpiler/passes/optimization/consolidate_blocks.py

```diff
@@ -55,15 +55,15 @@ def __init__(
     ):
         """ConsolidateBlocks initializer.
 
-        If `kak_basis_gate` is not `None` it will be used as the basis gate for KAK decomposition.
-        Otherwise, if `basis_gates` is not `None` a basis gate will be chosen from this list.
-        Otherwise the basis gate will be `CXGate`.
+        If `kak_basis_gate` is not ``None`` it will be used as the basis gate for KAK decomposition.
+        Otherwise, if ``basis_gates`` is not ``None`` a basis gate will be chosen from this list.
+        Otherwise, the basis gate will be :class:`.CXGate`.
```

Comment:
`kak_basis_gate` is still in italics here.

Reply:
good catch, fixed in e280530c5f278e087f320b62543c2cfee09ea4a7